app/src/main/java/com/example/infinite_track/presentation/screen/wfa/components/WfaRequestCard.kt

```
@@ -0,0 +1,240 @@
+
+@Composable
+private fun ScoreBadge(score: Double?, color: Color) {
+    Box(
+        modifier = Modifier
+            .size(42.dp)
+            .clip(CircleShape)
+            .background(color.copy(alpha = 0.11f)),
+        contentAlignment = Alignment.Center
+    ) {
+        Text(
+            text = score?.let { String.format("%.1f", it) } ?: "-",
```

detekt (1.23.8)

[warning] String.format("%.1f", it) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)